pkg/csidriver/driver.go

```diff
@@ -22,18 +22,19 @@ import (
 	"errors"
 	"fmt"
 	"io"
-	"io/ioutil"
-	"k8s.io/utils/mount"
 	"os"
 	"path/filepath"
 	"strings"
 	"sync"
+	"syscall"
 
 	corev1 "k8s.io/api/core/v1"
 	kerrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/klog/v2"
+	atomic "k8s.io/kubernetes/pkg/volume/util"
```

Comment:
I think that this should be something like "volumeutil", "kvolumeutil" or just "kutil" to be more descriptive to the package it is pulling in, not what feature of that package you are using.